tools/serverpod_cli/lib/src/analyzer/models/model_analyzer.dart

```diff
@@ -38,6 +38,7 @@ class SerializableModelAnalyzer {
   /// Best effort attempt to extract an model definition from a yaml file.
   static SerializableModelDefinition? extractModelDefinition(
     ModelSource modelSource,
+    GeneratorConfig config,
```

Comment:
Suggestion: only take the extraClasses as parameter.

If the extra classes is the only field that we need from the config I suggest we don't create a dependency towards the config here but instead make clear what this method needs by defining it as a parameter of the method.